### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by michaelwehrley**
* primary goal was to update `sass-rails` to `~> 6.0.0.beta1`
* Update other gems inorder to get bundle update to succeed

### CODE REVIEW

File: spec/dummy/config/initializers/assets.rb [hunk 1]
```diff
@@ -6,8 +6,11 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 Rails.application.config.assets.precompile += %w(
```
Comment (michaelwehrley) [inline spec/dummy/config/initializers/assets.rb]: @ericcf can you help me out here?  The update to sprockets had me change this file.  There has to be a better way to include these files.  In marigold I tried:
```ruby
Rails.application.config.assets.precompile += [
  "think_feel_do_engine.js",
  ...
  "social_networking/*.png",
  "*.png",
  "*.jpeg",
  "*.jpg",
  "*.gif",
  "*.ico"
]
```
but that doesn't work?  Thanks!

Comment (ericcf) [inline spec/dummy/config/initializers/assets.rb]: Check out my branch here: https://github.com/NU-CBITS/think_feel_do_engine/tree/v4.0

Specifically https://github.com/NU-CBITS/think_feel_do_engine/blob/v4.0/app/assets/config/manifest.js

Comment (ericcf) [inline spec/dummy/config/initializers/assets.rb]: Also take a look at moodtech: https://github.com/NU-CBITS/think_feel_do_so/tree/ecf_rails5

**Comment by ericcf** [comment]
2 legit

**Pull request merged**